enhancements/installer/vsphere-ipi-zonal-external-lb.md

```diff
@@ -0,0 +1,201 @@
+---
+title: vsphere-ipi-zonal-external-lb
```

Comment:
general thought:
The fact that there is a inter-dependency between IPI and onprem networking is a big limitation that we've been imposing on our customers. Hence I think having this enhancement is a great step forward. I do believe that it should be solved in a holistic manner - for all on prem platforms, rather than for vpshere only.
In assisted - the way we are categorizing it is - CMN (cluster managed netoworking) and UMN (user managed networking) - where, currently, it boils down to ipi/upi manifests (which define the networking yamls). Once this enhancement is implemented, we'll have to adjust our logic there.